Monopoly/Assets/Game.cs

```diff
@@ -106,4 +126,16 @@ void Update()
             }
         }
     }
+    void HandleRentPay(Property property, int payingPlayerId)
+    {
+        //TODO
+    }
+    void HandleStandingOnOwnPosition()
+    {
+        //JUST SHOW POPUP INFO?
```

Comment:
tu chyba przydałoby się w przyszłości podobne menu jak przy kupowaniu pola, tylko że dotyczące rozbudowy (o ile taką będziemy implementować)

Reply:
Ale rozbudowa jest możliwa zawsze przy ruchu gracza, nie trzeba stać na tym polu no nie?

Reply:
Ja zawsze grałem, że trzeba stać, to do uzgodnienia :)